app/src/main/res/layout/history_list_item.xml

```diff
@@ -11,7 +11,16 @@
 
     <include
         android:id="@+id/recently_closed_nav_empty"
-        layout="@layout/recently_closed_nav_item" />
+        layout="@layout/recently_closed_nav_item"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:visibility="visible"
+        android:layout_marginTop="8dp"
+        android:layout_marginBottom="0dp" />
```

Comment:
I think you only need `marginTop` here, but I am unsure if you need to make any changes here if we just added that 8dp Spacer and that's only if UX confirms that was intentional.

Reply:
same [case](https://github.com/mozilla-mobile/fenix/pull/24374#discussion_r847610255)